package/kernel/mac80211/patches/982-rt2x00-add-rf-self-txdc-calibration.patch

```diff
@@ -0,0 +1,89 @@
+--- a/drivers/net/wireless/ralink/rt2x00/rt2800lib.c
```

Comment:
Please create kernel patches like this `982-rt2x00-add-rf-self-txdc-calibration.patch` file using `git format-patch --to linux-wireless@vger.kernel.org --cc "Stanislaw Gruszka <sgruszka@redhat.com>" --cc "Kalle Valo <kvalo@codeaurora.org>"` with commits on top of the [wireless-drivers-next tree](https://git.kernel.org/pub/scm/linux/kernel/git/kvalo/wireless-drivers-next.git) so they end up in [the right patchwork](https://patchwork.kernel.org/project/linux-wireless/list/). (your current patches should apply on that tree just fine)
Each commit should include a meaningful patch description starting with the `rt2x00: ` prefix and your `Signed-of-by: ` tag so they are fit for upstream submission. A basic description like the ones you wrote for the commits adding the patches to OpenWrt is fine, just also make sure to mention that this is for MT7620/RT6352 WiSoC in each patch description.